dataland-frontend/src/components/general/CompanyInfoSheet.vue

```diff
@@ -41,9 +50,17 @@ const useMobileView = computed<boolean | undefined>(() => injectedMobileView?.va
 const sheet = ref<HTMLDivElement>();
 const attachedSheet = ref<HTMLDivElement>();
 
-const { companyId } = defineProps<{
-  companyId: string;
-}>();
+const { companyId, showSearchBar, showSingleDataRequestButton } = defineProps({
+  companyId: String,
```

Comment:
I would suggest to extend this to 

`companyId: {
   type: String,
   required: true,
},`

as my IDE otherwise complains at two occurrences in this file since the companyId is then understood to be of type `string | undefined`

Reply:
yes. I will adapt this

Reply:
done